lib/src/date-input/DatePicker.tsx

```diff
@@ -129,9 +135,8 @@ const HeaderButton = styled.button`
   svg {
     display: inline-block;
```

Comment:
You don't need to change the display of the SVG. Indeed, the only necessary rules are the width and height, user-select and flex-shrink are not doing anything at all...

Reply:
```
svg {
   width: 24px;
   height: 24px;
}
```